pkg/readiness/object_tracker.go

```diff
@@ -29,13 +30,16 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var readinessRetries = flag.Int("readiness-retries", 0, "The number of resource ingestion attempts allowed before the resource is disregarded")
```

Comment:
Note that a value of `-1` should mean "retry indefinitely"

Reply:
Added this to the flag usage string.